coalib/settings/ConfigurationGathering.py

```diff
@@ -135,27 +135,30 @@ def warn_nonexistent_targets(targets, sections, log_printer):
 
     # Can't be summarized as python will evaluate conditions lazily, those
     # functions have intended side effects though.
-    files_config_absent = warn_config_absent(sections, 'files', log_printer)
-    bears_config_absent = warn_config_absent(sections, 'bears', log_printer)
+    files_config_absent = warn_config_absent(sections, ['files'], log_printer)
+    bears_config_absent = warn_config_absent(sections, ['bears', 'aspects'],
+                                             log_printer)
     if files_config_absent or bears_config_absent:
         raise SystemExit(2)  # Invalid CLI options provided
 
 
-def warn_config_absent(sections, argument, log_printer):
+def warn_config_absent(sections, arguments, log_printer):
```

Comment:
this is a breaking change.
The parameter name cant change, and the function must accept and correctly handle either a str or list of str